vcf_stuff/filtering/annotate_somatic_vcf.smk

```diff
@@ -255,10 +200,11 @@ rule somatic_vcf_prep:
         t_name_arg = f"-tn {params.t_name}" if params.t_name else ""
         n_name_arg = f"-nn {params.n_name}" if params.n_name else ""
         r_name_arg = f"-rn {params.r_name}" if params.r_name else ""
-        shell(f'pcgr_prep {t_name_arg} {n_name_arg} {r_name_arg} {input.vcf}' 
-              f' | bgzip -c > {output.vcf} && tabix -f -p vcf {output.vcf}')
+        shell(f'pcgr_prep {t_name_arg} {n_name_arg} {r_name_arg} {input.vcf} | '
+              f'bgzip -c > {output.vcf} && tabix -f -p vcf {output.vcf}')
 
 if GENOME != 'GRCh37':
```

Comment:
One for the todo list - we should probably error out right at the start for `GENOME != 'hg38'`.